bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigOptionProvider.java

```diff
@@ -28,8 +28,7 @@
      *            the parameter name for which the requested options shall be returned
      * @param locale
      *            locale
-     * @return the configuration options provided by this provider (not
-     *         null, could be empty)
+     * @return the configuration options provided by this provider if any (may be null)
```

Comment:
What is the value of differentiating it? 
I simply adapted to JavaDoc do explicitly allow returning null, i.e. to indicate that the caller is able to handle it.